python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -31,36 +29,78 @@
 def getDiracEnv(force=False):
     """
     Returns the dirac environment stored in a global dictionary by Ganga.
-    This is expected to be stored as some form of 'source'-able file on disk which can be used to get the printenv after sourcing
-    Once loaded and stored his is used for executing all DIRAC code in future
+    Once loaded and stored this is used for executing all DIRAC code in future
+
     Args:
         force (bool): This triggers a compulsory reload of the env from disk
     """
     global DIRAC_ENV
     with Dirac_Env_Lock:
-        if DIRAC_ENV == {} or force:
-            config_file = getConfig('DIRAC')['DiracEnvFile']
-            if not os.path.exists(config_file):
-                absolute_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../..', config_file)
+        if not DIRAC_ENV or force:
```

Comment:
I wonder if we should be overloading the 'force' method here. Or even drop it.
I've only ever used it in testing and I don't think it's used anywhere in the codebase?.
I worry that for LHCb users this could cause problems if we allow it to remain and then it gets used somewhere because it works on vanilla DIRAC.